air/src/air/mod.rs

```diff
@@ -303,15 +305,13 @@ pub trait Air: Send + Sync {
         Vec::new()
     }
 
-    // AUXILIARY PROOF VERIFIER
+    // LOGUP-GKR EVALUATOR
     // --------------------------------------------------------------------------------------------
 
-    /// Returns the [`GkrVerifier`] to be used to verify the GKR proof.
-    ///
-    /// Leave unimplemented if the `Air` doesn't use a GKR proof.
-    fn get_gkr_proof_verifier<E: FieldElement<BaseField = Self::BaseField>>(
+    /// Returns the object needed for the LogUp-GKR argument.
+    fn get_logup_gkr_evaluator<E: FieldElement<BaseField = Self::BaseField>>(
         &self,
-    ) -> Self::GkrVerifier {
+    ) -> Self::LogUpGkrEvaluator {
         unimplemented!("`get_auxiliary_proof_verifier()` must be implemented when the proof contains a GKR proof");
```

Comment:
This message needs to be updated with the new method name

Reply:
fixed